.env.example

```diff
@@ -1,9 +1,3 @@
-JWKS_URI=https://citadel.demo.aserto.com/dex/keys
```

Comment:
Just curious, why remove these? All the other todo samples have them, seems better to stay consistent?

I think Roie's theory was that you could perhaps hook this up to your own IDP by switching these environment variables. I'm not sure that's ever been tested, but I think it's reasonable to keep these environment variables.

Reply:
I removed them because in the Go app they are no longer required. These variables can still be set in the environment or in the .env file but the app also has default values.
But I see your point and will put them back in .env.example as a form of documentation.